Sources/VaporAPNS/Exports.swift

```diff
@@ -0,0 +1,6 @@
+#if !BUILDING_DOCC
+
+@_exported import APNS
+@_exported import APNSCore
```

Comment:
If we're releasing a new version of APNS we should remove all exports - they just cause problems. It would be interesting to see what is required from these exports for 'normal' usage of the library